src/components/Triangle/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+
+const Triangle = ({ row, col, values, appConfig }) => {
+
+  const { side, height, styleTarsiaTriangle, triangleYTextPadding, styleTarsiaText } = appConfig;
+  const orientation = ((col + row) % 2 === 0) ? "down" : "up"
+  const rotate = orientation === "up" ? 180 : 0
+  const translateX = (col-1) * side / 2
+  const translateY = (row-1) * height
+  const transform = `translate(${translateX},${translateY}) rotate(${rotate} ${side / 2},${height/2})`
```

Comment:
Used template literals to avoid having to manually concatenate the values here and in other places in the component: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals